ReactiveArchitecture/Classes/nowplaying/viewcontroller/NowPlayingTableViewController.swift

```diff
@@ -44,12 +44,12 @@ class NowPlayingTableViewController: UITableViewController {
      Add passed in list to table view with animation for each entry to add
      Parameters: listToAdd - list to add
     */
-    func addAll(listToAdd: Array<MovieViewInfo>!) -> Void {
```

Comment:
`-> Void` is implicit. This (and a lot of spacing and implicit return stuff) was fixed with 1 pass of `swiftlint autocorrect` 💯

Reply:
In the beginning I didn't have the void on my functions, then started putting it in. No idea why..

:thumbsup: